src/dls_bluesky_core/tasks/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .wrapped import move, move_relative, set_absolute, set_relative, sleep, wait
+
+"""
+This package is intended to hold MsgGenerator functions which simultaneously:
+- Contain functionality which is useful to expose to BlueAPI
+- Do not collect data so matches closer to Bluesky 'plan_stubs' than 'plans'
+- May not be useful for composing plans (functionality may simply be calling a stub)
```

Comment:
I feel we are structuring this repo _for_ blueapi a bit too much, how about instead putting these plans in `stubs` and having a single module for blueapi compatibility called `tasks.py` which is just

```python
"""
This module exists to easily import MsgGenerator functions which simultaneously:
- Contain functionality which is useful to expose to BlueAPI
- Do not collect data so matches closer to Bluesky 'plan_stubs' than 'plans'
- May not be useful for composing plans (functionality may simply be calling a stub) 
"""

from dls_bluesky_core.stubs import (  #noqa:  F401
    set_absolute, 
    set_relative, 
    move, 
    move_relative, 
    sleep, 
    wait) 
```

Reply:
I'm also not sold on the name `tasks` just because it's so overloaded. Maybe `support_tasks`?

Reply:
Note: I am heavily on the fence about all of the above! Discussion encouraged!

Reply:
Also, we're not sure tasks is a good name as it means something already in the Blueapi context

Reply:
Removed references to BlueAPI, just vague concept of "services that run plans". Tasks merged back into Stubs with explicit __exporting__ when want only a subset of "plans" available from module/to have requirements enforced